themes/bootstrap5/templates/search/advanced/layout.phtml

```diff
@@ -229,7 +237,7 @@
 </div>
 <?php
 $script = <<<JS
-    $('#groupPlaceHolder').click(function(e) {
+    $('#adv-add-group-btn').click(function(e) {
```

Comment:
Is it worth de-jQuerying this while we're changing it anyway, since I think it would be a pretty minor change?